### PR TITLE
Added --force (-n) option. Fixes #1059

### LINKED ISSUE #1059: --force option for CLI
`tslint --force` should return a status code of 0 even if there are lint failures.

follow-up from https://github.com/palantir/tslint/issues/1053

### PR DESCRIPTION
**Pull request by sasidhar**
Fixes #1059

### CODE REVIEW

**Comment by JKillian** [comment]
Thanks @sasidhar! Is there any specific reason you chose `-n` as an alias for `--force`? I'm thinking we should just have `--force` and not bother with a shorthand `-n` option.

File: src/tslint-cli.ts [hunk 1]
```diff
@@ -46,6 +46,10 @@ let processed = optimist
             alias: "config",
             describe: "configuration file",
         },
+        "n": {
+            alias: "force",
+            describe: "return status code 0 even if there are lint errors",
+        },
```
Comment (JKillian) [inline src/tslint-cli.ts]: Let's add `type: "boolean"` here

**Comment by sasidhar** [comment]
@JKillian Sure. I will update and send another PR.

**Comment by JKillian** [comment]
No worries @sasidhar, I had a spare moment so I just made the tweaks myself over in https://github.com/palantir/tslint/pull/1073. Thanks for this PR, this'll be a useful feature!

**Pull request merged**